fabric-data-attachment-api-v1/src/main/java/net/fabricmc/fabric/api/attachment/v1/AttachmentType.java

```diff
@@ -93,6 +93,11 @@ default boolean isPersistent() {
 	@Nullable
 	Supplier<A> initializer();
 
+	/**
+	 * @return whether this attachment type can be synchronized with clients
+	 */
+	boolean isSynced();
```

Comment:
should this be `isSyncable`? to me, `isSynced` represents the current state of the attachment rather than its properties

Reply:
`isSynced` is fine. It's also used for synced dynamic registries:
https://github.com/FabricMC/fabric/blob/3b9245a97679f6c58089dd803021f93d91402b29/fabric-registry-sync-v0/src/main/java/net/fabricmc/fabric/api/event/registry/DynamicRegistries.java#L142-L145
(including in [Yarn](<https://github.com/FabricMC/yarn/blob/790c144326c804dfeb2438957bfe0a9982411d59/mappings/net/minecraft/registry/RegistryLoader.mapping#L5>))